Resources/Prototypes/Entities/Structures/Shuttles/thrusters.yml

```diff
@@ -42,9 +44,55 @@
   placement:
     mode: SnapgridCenter
 
+- type: entity
+  id: BaseThrusterStatic  # for when we dont want thrusters to be moveable such as pods and arrivals.
+  parent: BaseStructure
+  name: stationary thruster
+  description: A stationary thruster that allows a shuttle to move.
+  abstract: true
+  components:
+    - type: AmbientSound
```

Comment:
This just looks like a duplicate of BaseThruster, please reduce the duplication by either using composition or inheritance.

Reply:
Hmm. Okay yeah. Let me think of how to do that without causing more parent spaghetti.